src/libs/notificationService.js

```diff
@@ -15,7 +15,12 @@ export const NotificationService = {
    */
   getBanners: async () => {
     const env = await Config.getEnv();
-    return $.getJSON(gcs + '/' + env + '_' + bannerFileName, (data) => {
+    const hash = await Config.getHash();
+    const url =
+      env === 'local'
+        ? `/broad-duos-banner/${hash}_${bannerFileName}`
+        : gcs + '/' + env + '_' + bannerFileName;
+    return $.getJSON(url, (data) => {
```

Comment:
Optional ... the fact that this url construction was in `NotificationService` is a bit of an outlier and older code. It would be nice if this were consolidated somewhere else, like `ajax.js` or `config.js`. Definitely not a PR blocker.